specifications/barriers/README.md

```diff
@@ -0,0 +1,17 @@
+# Barrier synchronization
+-
+
+## Barriers.tla
+
+A specification of a Reusable Barrier synchronization facility using as
+presented in the INFO09012 Parallel Programming course in ULiège by 
+Prof. Pascal Fontaine.
+
+A refinement towards an abstract Barrier specification is proven with TLAPS.
+
+The main invariant `Inv` is verified in the provided model for $N = 7$ 
+processes.
+
+## Barrier.tla
+
+A specification of an abstract Barrier. No model is provided.
```

Comment:
Wouldn't it make sense to have TLC check a basic property of that specification as well?